packages/zoe/tools/manualTimer.js

```diff
@@ -1,37 +1,79 @@
+// @ts-check
+
 import { E } from '@agoric/eventual-send';
+import { makeStore } from '@agoric/store';
+
+import './types';
+
+/* eslint-disable jsdoc/valid-types */
```

Comment:
Can we add a comment for why this is disabled? I don't immediately see what the problem is

Reply:
`(msg?: string)` passes Typescript, but causes an eslint-jsdoc error.  `(msg: string=)` passes eslint-jsdoc, but causes a Typescript error.

I'll add a comment.

Reply:
Better.  I broke out a `@callback ManualTimerTick` definition and it passes both.